lib/pure/unittest.nim

```diff
@@ -185,7 +192,15 @@ proc defaultConsoleFormatter*(): ConsoleOutputFormatter =
     # Reading settings
     # On a terminal this branch is executed
     var envOutLvl = os.getEnv("NIMTEST_OUTPUT_LVL").string
-    var colorOutput  = not existsEnv("NIMTEST_NO_COLOR")
+    var colorOutput  = isatty(stdout)
+    if existsEnv("NIMTEST_COLOR"):
```

Comment:
This ``existsEnv`` check is not required, ``getenv`` returns ``""`` if it doesn't exist. And yes, I do like that because it simplifies the logic.

Reply:
My original thought was that, if ``NIMTEST_COLOR`` is defined, even if not what we recognize (neither ``never`` nor ``always``), we would ignore ``NIMTEST_NO_COLOR``.

Perhaps you have a better idea of introducing a new env variable and deprecating the old?